### PR TITLE
Ontology Update

### PR DESCRIPTION
**Pull request by oxisto**
- Added generated OpenAPI defs for ontology
- Added metadata for required for discovery service

### CODE REVIEW

File: api/discovery/experimental.proto [hunk 1]
```diff
@@ -68,13 +69,13 @@ message ListGraphEdgesRequest {
...
+  repeated GraphEdge edges = 1 [(google.api.field_behavior) = REQUIRED];
   string next_page_token = 2;
 }
 
 message GraphEdge {
-  string id = 1;
-  string source = 2;
-  string target = 3;
-  string type = 4;
+  string id = 1 [(google.api.field_behavior) = REQUIRED];
```
Comment (anatheka) [inline api/discovery/experimental.proto:77]: What is about `(buf.validate.field).string.min_len = 1,`, `(buf.validate.field).string.uuid = true,`, ...?

File: api/ontology/service.proto [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+package clouditor.ontology.v1;
+
+import "api/ontology/ontology.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+
+option go_package = "clouditor.io/clouditor/v2/api/ontology";
+
+service OntologyService {
```
Comment (anatheka) [inline api/ontology/service.proto:36]: Did you forget to add the implementation?

**Review by anatheka — COMMENTED** [review]
Only some minor things.

File: api/ontology/service.proto (hunk 2 above)
Comment (oxisto) [reply api/ontology/service.proto:36]: No, this was intentionally done to provoke openapi in generating all the resources. I remember you had a similar approach with a fake service.

File: api/discovery/experimental.proto (hunk 1 above)
Comment (oxisto) [reply api/discovery/experimental.proto:77]: The IDs here are not UUIDs but are the resource IDs (which are probably URLs), but min length makes sense

**Review by anatheka — APPROVED** [review]
LGTM

**Pull request merged**